packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import { useRouter } from "next/navigation";
+
+import { Button } from "react-aria-components";
+
+import { WalkthroughInfo } from "../../../data/src/useWalkthroughData";
+
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps {
+  id: string;
+  data: WalkthroughInfo;
+  href?: string;
+}
+
+export default function WalkthroughCard({
+  id,
+  data,
+  href,
+}: WalkthroughCardProps) {
+  const router = useRouter();
+
+  const handleClick = () => {
+    if (href) router.push(href);
+  };
+
+  const canDisplayCard =
+    data && data.title && data.subtitle && data.description;
+
+  if (!canDisplayCard) return null;
+
+  return (
+    <Button
```

Comment:
I think semantically this should be an `<a>` tag and not a button as it's sole purpose is to navigate via an href.